.github/workflows/apt-arm-packages.yaml

```diff
@@ -1,13 +1,11 @@
 # Test installing our ubuntu and debian ARM64 packages for the latest version.
 name: APT ARM64 packages
 on:
-  schedule:
-    # run daily 0:00 on main branch
-    - cron: '0 0 * * *'
   push:
     tags:
     - '*'
     branches:
+    - main
     - release_test
```

Comment:
What's that, by the way? We don't have the 'release_test' branch.

Reply:
i guess this was supposed to be used to trigger all the package tests but we never started using it